third_party/terraform/resources/resource_google_project_services.go

```diff
@@ -355,3 +220,43 @@ func expandServiceUsageProjectServicesServices(v interface{}, d TerraformResourc
 	}
 	return convertStringArr(v.(*schema.Set).List()), nil
 }
+
+// Retrieve a project's services from the API
+func listCurrentlyEnabledServices(project string, config *Config, timeout time.Duration) (map[string]struct{}, error) {
```

Comment:
We expect this to move within the same file?

Reply:
Ah, I moved it and then moved it back, and mistargeted where it was supposed to go.  NBD either way.